althea/chain.json

```diff
@@ -45,12 +45,12 @@
   ],
   "codebase": {
     "git_repo": "https://github.com/AltheaFoundation/althea-L1",
-    "recommended_version": "v1.3.0",
+    "recommended_version": "v1.5.1",
```

Comment:
Please register this version into the versions.json file.

Reply:
Done